internal/venice-common/src/main/java/com/linkedin/venice/pubsub/api/PubsubProduceResult.java

```diff
@@ -0,0 +1,33 @@
+package com.linkedin.venice.pubsub.api;
+
+/**
+ * An interface implemented by specific PubsubProducerAdapters to return the result of a produce action.
+ */
+public interface PubsubProduceResult {
+  /**
+   * The offset of the record in the topic/partition.
+   */
+  long offset();
+
+  /**
+   * The size of the serialized, uncompressed key in bytes. If key is null, the returned size
+   * is -1.
+   */
+  int serializedKeySize();
+
+  /**
+   * The size of the serialized, uncompressed value in bytes. If value is null, the returned
+   * size is -1.
+   */
+  int serializedValueSize();
```

Comment:
No need for both of these since we only ever use the sum of the two. So we can make the interface suit our needs, and have just a single `getSerializedSize()`.